src/test/kotlin/io/snyk/plugin/net/CliConfigServiceTest.kt

```diff
@@ -0,0 +1,71 @@
+package io.snyk.plugin.net
+
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.MockWebServer
+import okio.buffer
+import okio.source
+import org.hamcrest.MatcherAssert.assertThat
+import org.hamcrest.core.IsEqual.equalTo
+import org.hamcrest.core.IsNull.nullValue
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import snyk.net.HttpClient
+import java.nio.charset.StandardCharsets
+
+class CliConfigServiceTest {
+    @JvmField
+    @Rule
+    val server: MockWebServer = MockWebServer()
```

Comment:
Love usage of an http server mock.

Reply:
We should use that for false-positive and cli download and all the rest, too ;)

Reply:
si, senior :).